### PR TITLE
Removing all know exif values

### PR DESCRIPTION
**Pull request by ArgiesDario**
Here is the code I've mentioned, a new method called "delete_all", which deletes all known exif tracking values

### CODE REVIEW

File: exif/_image.py [hunk 1]
```diff
@@ -110,6 +110,11 @@ def delete(self, attribute):
         """
         self.__delattr__(attribute)
 
+    def delete_all(self):
+        """Remove all the known EXIF tracking values, like GPS, DATETIME, and others
+        """
+        [self.__delattr__(e) for e in dir(self) if e in REMOVAL_EXIF]
```
Comment (TNThieding) [inline exif/_image.py:116]: Could you please explain how you chose the tags that you included in `REMOVAL_EXIF`? Would it be okay for your use case to simply remove all EXIF tags like what I have below?
`[self.__delattr__(tag) for tag in self._segments['APP1'].get_tag_list()]`

Comment (ArgiesDario) [reply exif/_image.py:116]: I choose the tags by picking the ones that had "sensible" info about the picture with several .jgp I have with a lot of exif data I have, then saving it and running a tool that checks the EXIF data you have in an image, until I saw there is no trace of info on it

At the begining I did something related to your approach, clearing everything up, but when I saved back the image with the cleared up EXIF data, and the opening it again with this package, I was facing this issue

```
Traceback (most recent call last):
  File "test.py", line 42, in <module>
    my_image = Image(image_file)
  File "/home/kali/Desktop/metadata_test/repo/exif/exif/_image.py", line 67, in __init__
    self._parse_segments(img_hex)
  File "/home/kali/Desktop/metadata_test/repo/exif/exif/_image.py", line 53, in _parse_segments
    self._segments['APP1'] = App1MetaData(img_hex[app1_start_index:cursor])
  File "/home/kali/Desktop/metadata_test/repo/exif/exif/_app1_metadata.py", line 227, in __init__
    self._unpack_ifd_tags()
  File "/home/kali/Desktop/metadata_test/repo/exif/exif/_app1_metadata.py", line 118, in _unpack_ifd_tags
    self._segment_hex.set_endianness(self._segment_hex.read(cursor, 2))
  File "/home/kali/Desktop/metadata_test/repo/exif/exif/_hex_interface.py", line 115, in set_endianness
    assert endian_constant in [EXIF_BIG_ENDIAN_HEADER, EXIF_LITTLE_ENDIAN_HEADER]
AssertionError
```

Looks like files with cleared up EXIF are having problems to import them again with this package. I did not want to get involved into it (and did not understood much on why is it failing) so I've created a subset of EXIF values, only containing sensible ones

If you can fix this issue of re-importing EXIF cleared up images I would love to go with your approach, it is much simpler and without repeated tags in _constants, but I need a way to: clear, save, reopen and be able to check that everything is empty (and if the attribute "has_exif" is False after clear up, much better)

Thank you!

Comment (TNThieding) [reply exif/_image.py:116]: Could you please provide a sample image where deleting all tags and re-opening the image causes an `AssertionError`? In all the cases I tried, I was able to delete all tags, save the image, and re-open it just fine.

Comment (ArgiesDario) [reply exif/_image.py:116]: Sure, here you have 2 images I've tried with. I've downloaded them from samples with a lot of exif data. I do not know if uploading here will clear some data, if not let me know I can send those somewhere else

corrupted
GPS

Comment (TNThieding) [reply exif/_image.py:116]: Thank you for providing these samples. Using the method I proposed above, I was able to reopen both images after deleting all tags. The only three tags that remain are `x_resolution`, `y_resolution`, and `resolution_unit` from the TIFF data.

Comment (TNThieding) [reply exif/_image.py:116]: Could you please try my code above with some of your sample images to see if it works or if it raises the exceptions you saw before?

**Comment by TNThieding** [comment]
Thanks for checking over my changes and for the contribution. I just released it in v0.11.0 of the package, which is now live. I'm therefore closing out this pull request.

**Pull request closed without merging**